runtime/service/src/main/java/org/apache/polaris/service/admin/PolarisServiceImpl.java

```diff
@@ -170,6 +171,18 @@ public Response createCatalog(
     return Response.status(Response.Status.CREATED).build();
   }
 
+  private void validateClientId(String clientId) {
+    if (!clientId.matches("^[0-9a-f]{16}$")) {
```

Comment:
Why do we only allow `[0-9a-f]`?

Reply:
I believe, that matches current client ID values as generated by Polaris.

We can certainly change validation rules later. I do not think it is critical for this PR.